Fehlerbeseitigungen

Illumnus opened the pull request.

Apollon77 commented:
Thank you. 

Can you tell more details which errors are fixed by ignoring some messages ?

And please exclude your vs directory from git and also remove from this pr. Thank you.

Illumnus commented:
Sorry, in am really new in GitHub.. :-(

I fixed the following Errors In these Events: 
ShutterContactStateReceived,ThermostatStateReceived, PushButtonStateReceived

If there was a already paired but new Device.
It was created without the "Type" "value"
it was created with a "SRC" and "SERIAL" in lowercharacters
so it could not be found in the devices list because of a conversion to UpperCharacters
and never received DATA

AND
in the normal Message from a Device there is no RAW part.
so i removed the extraction of the raw information for the device src.
Because it was resulting in different device Names, some devices have only there ID as SRC and Name and if i start a new pairing the Device gets a complete different SRC and Name

ALSO
in the event "ShutterContactStateReceived" 
was a createButton(data); at the End instead of the Correct createContact(data);

AT LEAST
in CreateContact i changed the Text from "Button" to "Window Contact"

Hope this Helps

Thank You
 Very much

Apollon77 commented:
Great details. Thank you

Apollon77 commented:
Ps: only question ... is there an open pr or alle closed now?
